plotting/plot_factory/tests/test_layout.py

```diff
@@ -0,0 +1,55 @@
+# ############################################################################### #
+# Autoreduction Repository : https://github.com/ISISScientificComputing/autoreduce
+#
+# Copyright &copy; 2020 ISIS Rutherford Appleton Laboratory UKRI
+# SPDX - License - Identifier: GPL-3.0-or-later
+# ############################################################################### #
+"""
+Unit tests for plot factory Layout
+"""
+
+# Core Dependencies
+import unittest
+from mock import patch
+
+
+# Internal Dependencies
+from plotting.plot_factory.layout import Layout
```

Comment:
Nit: We could add these imports to the `__init__.py` of the `plotting.plot_factory` module. This would reduce the import lines to:
`from plotting.plot_factory import Layout`

example of this is [here](https://github.com/ISISScientificComputing/autoreduce/pull/539/files#diff-7c738d984b2bdb58f92bb344f69d3751R1-R11)